### DISCUSSION TITLE
Why the database list in Dev and Prod are not the same?

### DISCUSSION THREAD

**Discussion by agoncal** [jhipster/generator-jhipster-quarkus#212] [Q&A]
When generating a project, we are asked about dev and prod databases. But the lists are not the same. Production has several ones (MySQL, MariaDB, PostgreSQL, etc.) and dev much less.

With DevServices in Quarkus 2, does it make sense ? Shouldn't the dev list be the same as prod (+ H2 disk and in-mem) ?

```
? Which *production* database would you like to use? (Use arrow keys)
❯ MySQL
  MariaDB
  PostgreSQL
  Oracle
  Microsoft SQL Server
  
? Which *development* database would you like to use? (Use arrow keys)
❯ H2 with disk-based persistence
  H2 with in-memory persistence
  MySQL  
```

**Comment by vishal423** [comment]
The development database list is a subset filtered by the chosen production database. In your case, since you selected MySQL as the production database, you have the option to choose MySQL or H2. Allowing to choose another development database like PostgreSQL, or Oracle doesn't make sense.
